src/rules/rdf/mustNotExist.js

```diff
@@ -4,7 +4,7 @@ import * as messages from 'messages';
 
 
 export function mustNotExist(xmlDoc, filename=null,
-                             namespace='http://www.mozilla.org/2004/em-rdf#') {
+    {namespace='http://www.mozilla.org/2004/em-rdf#'}={}) {
```

Comment:
It does, however in a previous comment you said 
> Shouldn't this default to the namspace previously set as a default?

so I assumed that namespace should have a default value?